fastcrypto/src/secp256r1/mod.rs

```diff
@@ -408,21 +408,7 @@ impl Signer<Secp256r1Signature> for Secp256r1KeyPair {
 
 impl From<Secp256r1PrivateKey> for Secp256r1KeyPair {
     fn from(secret: Secp256r1PrivateKey) -> Self {
-        let name = Secp256r1PublicKey::from(&secret);
-        Secp256r1KeyPair { name, secret }
-    }
-}
-
-impl zeroize::Zeroize for Secp256r1KeyPair {
```

Comment:
What is the main reason of not zeroizing the keypair anymore?

Reply:
Drop of the private key takes care of that already